dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -376,4 +428,293 @@ public void zipTempDirectoryToStream(OutputStream out) throws IOException {
+			_list = APILocator.getUserAPI().findAllUsers();
+			_list.add(APILocator.getUserAPI().getDefaultUser());
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator +  User.class.getName() + ".xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			DotConnect dc = new DotConnect();
+
+			/* counter */
+			dc.setSQL("select * from counter");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator + "Counter.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			/* counter */
+			dc.setSQL("select * from address");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator + "Address.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+
+			/* image */
+			_list = ImageLocalManagerUtil.getImages();
+
+			/*
+			 * The changes in this part were made for Oracle databases. Oracle has problems when
+			 * getString() method is called on a LONG field on an Oracle database. Because of this,
+			 * the object is loaded from liferay and DotConnect is not used
+			 * http://jira.dotmarketing.net/browse/DOTCMS-1911
+			 */
+
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator + "Image.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			/* portlet */
+
+			/*
+			 * The changes in this part were made for Oracle databases. Oracle has problems when
+			 * getString() method is called on a LONG field on an Oracle database. Because of this,
+			 * the object is loaded from liferay and DotConnect is not used
+			 * http://jira.dotmarketing.net/browse/DOTCMS-1911
+			 */
+			dc.setSQL("select * from portlet");
+			_list = dc.getResults();
+			_xstream = new XStream(new DomDriver());
+			_writing = new File(backupTempFilePath + File.separator + "Portlet.xml");
+			_bout = new BufferedOutputStream(Files.newOutputStream(_writing.toPath()));
+			try {
+				_xstream.toXML(_list, _bout);
+			} finally {
+				CloseUtils.closeQuietly(_bout);
+			}
+			_list = null;
+			_bout = null;
+
+			/* portlet_preferences */
+
+			try{
+				_list = PortletPreferencesLocalManagerUtil.getPreferences();
+			}catch(Exception e){

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)